Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -3404,6 +3404,7 @@ def main():  # pragma: no cover
     elif command == 'splunk-submit-event':
         splunk_submit_event_command(service, args)
     elif command == 'splunk-notable-event-edit':
+        base_url = f"https://{connection_args['host']}:{connection_args['port']}/"
```

Comment:
Why assign the base_url just for this command?

Reply:
Because it is the only command that uses it. See @ilappe [comment](https://github.com/demisto/content/pull/38375#discussion_r1948059389).